Packages/io.chainsafe.web3-unity/Runtime/Scripts/EVM/Token/Erc721.cs

```diff
@@ -133,5 +148,46 @@ public static async Task<string> URI(Web3 web3, string contractAddress, string t
             });
             return contractData[0].ToString();
         }
+        
+        /// <summary>
+        /// Mints ERC721 token
+        /// </summary>
+        /// <param name="web3"></param>
+        /// <param name="abi"></param>
+        /// <param name="contractAddress"></param>
+        /// <param name="uri"></param>
+        /// <returns></returns>
+        public static async Task<object[]> MintErc721(Web3 web3, string abi, string contractAddress, string uri)
+        {
+            const string method = "safeMint";
+            var destination = await web3.Signer.GetAddress();
+            var contract = web3.ContractBuilder.Build(abi, contractAddress);
+            return await contract.Send(method, new object[] { destination, uri });
+        }
+        
+        /// <summary>
+        /// Transfers ERC721 token
+        /// </summary>
+        /// <param name="web3"></param>
+        /// <param name="contractAddress"></param>
+        /// <param name="toAccount"></param>
+        /// <param name="tokenId"></param>
+        /// <returns></returns>
+        public static async Task<object[]> TransferErc721(Web3 web3, string contractAddress, string toAccount, int tokenId)
```

Comment:
Token ID's should be in string format as they aren't used in calculations but only to identify. Additionally, Large token ID's like CS's ERC1155 tokens will overflow

Reply:
updated